aries-site/src/examples/templates/forms/SingleFormFieldExample.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Text,
+  TextInput,
+} from 'grommet';
+
+export const SingleFormFieldExample = () => {
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  return (
+    <Box
+      pad="medium"
+      round="small"
+      elevation="medium"
+      gap="medium"
+      width="medium"
+    >
+      <Header
+        direction="column"
+        align="start"
+        gap="xxsmall"
+        pad={{ horizontal: 'xxsmall' }}
+      >
+        {/* Use semantically correct heading level and adjust size as 
+        needed. In this instance, this example is presented within an 
+        HTML section element and this is the first heading within the 
+        section, therefor h2 is the semantically correct heading. For 
+        additional detail, see https://design-system.hpe.design/foundation/typography#semantic-usage-of-heading-levels). */}
+        <Heading size='small' level={2} margin="none">
+          Delete
```

Comment:
```suggestion
          Delete Cluster
```
This is a design question, but should we be more specific in the title. I felt like I was wondering "What am I deleting?"

Reply:
good point let me think on that